manage_breast_screening/config/jinja2_env.py

```diff
@@ -1,28 +1,63 @@
+import re
+
+import arrow
 from django.templatetags.static import static
 from django.urls import reverse
 from jinja2 import ChoiceLoader, Environment, PackageLoader
 from markupsafe import Markup, escape
 
 
 def no_wrap(value):
+    """
+    Wrap a string in a span with class app-no-wrap
+
+    >>> no_wrap('a really long string')
+    Markup('<span class="app-nowrap">a really long string</span>')
+    """
     return Markup(f'<span class="app-nowrap">{escape(value)}</span>' if value else "")
 
 
 def as_hint(value):
+    """
+    Wrap a string in a span with class app-text-grey
+
+    >>> as_hint('Not provided')
+    Markup('<span class="app-text-grey">Not provided</span>')
+    """
     return Markup(f'<span class="app-text-grey">{value}</span>' if value else "")
 
 
+def sentence_case(value):
+    """
+    Capitalise the first word of a sentence.
+
+    >>> sentence_case('a quick brown fox jumps over the lazy dog')
+    'A quick brown fox jumps over the lazy dog'
+
+    Unlike the built in `capitalize` filter, this will preserve
+    capital letters already in the string:
+
+    >>> sentence_case('not in PACS')
+    'Not in PACS'
+    """
+    if not value:
+        return ""
+
+    return value[0].upper() + value[1:]
+
+
 def format_words(value, separator="_"):
```

Comment:
Aside: I was never happy with this filter, or at least the naming of it, which didn't feel intuitive.

Reply:
I've just realised we don't even need this, there is a more conventional way to accomplish the same thing in Django.